pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Looks like you are using the wrong pnpm version.

Try `corepack enable pnpm` to select the correct version.

Also see https://styfle.dev/blog/corepack